image.yaml

```diff
@@ -181,61 +200,41 @@ envs:
     - name: NO_PROXY
       description: A comman separated lists of hosts, IP addresses or domains that can be accessed directly. This will be used for both Maven builds and Java runtime.
       example: "foo.example.com,bar.example.com"
-labels:
-    - name: "io.openshift.s2i.scripts-url"
-      value: "image:///usr/local/s2i"
-    - name: io.fabric8.s2i.version.maven
-      value: "3.3.9-2.8"
-    - name: io.fabric8.s2i.version.jolokia
-      value: "1.3.6"
-    - name: io.k8s.description
-      value: "Platform for building and running plain Java applications (fat-jar and flat classpath)"
-    - name: io.k8s.display-name
-      value: "Java Applications"
-    - name: io.openshift.tags
-      value: "builder,java"
-    - name: io.openshift.s2i.destination
-      value: "/tmp"
-    - name: org.jboss.deployments-dir
-      value: "/deployments"
-packages:
-    - rh-maven33
-    - PyYAML
-# Use the run script as default since we are working as an hybrid image which can be
-# used directly to. (If we were a plain s2i image we would print the usage info here)
-cmd:
-    - "/usr/local/s2i/run"
 ports:
     - value: 8080
     - value: 8443
     - value: 8778
-sources:
-    - artifact: jolokia-jvm-1.3.6.redhat-1-agent.jar
+modules:
+      repositories:
+          - git:
+                  url: https://github.com/jboss-openshift/cct_module.git
+                  ref: master
+      install:
+          - name: s2i-common
+          - name: os-java-misc
+          - name: os-java-s2i
+          - name: os-java-jolokia
+          - name: os-java-hawkular
+          - name: os-java-run
+packages:
+      repositories:
+          - jboss-os
+          - jboss-rhscl
+      install:
+          - rh-maven33
+          - PyYAML
+# Use the run script as default since we are working as an hybrid image which can be
+# used directly to. (If we were a plain s2i image we would print the usage info here)
```

Comment:
Location of this comment changed and does not makes sense. Please move it to the `cmd` section.